src/cfd/screens/CFDCompareAccounts/CompareAccountsPlatformLabel.tsx

```diff
@@ -0,0 +1,46 @@
+import { twMerge } from 'tailwind-merge';
+
+import { Text } from '@deriv-com/ui';
+
+import { TPlatforms } from '@/types';
+
+import {
+    CompareAccountsPlatformLabelClass,
+    CompareAccountsPlatformLabelTextColorClass,
+    TCompareAccountsPlatformLabelClassProps,
+    TCompareAccountsPlatformLabelTextClassProps,
+} from './CompareAccounts.classnames';
+import { getPlatformType } from './CompareAccountsConfig';
+import { platformLabel } from './constants';
+
+type TCompareAccountsPlatformLabel = {
+    platform: TPlatforms.All;
+};
+
+const CompareAccountsPlatformLabel = ({ platform }: TCompareAccountsPlatformLabel) => {
+    const platformType = getPlatformType(platform);
+
+    return (
+        <div
+            className={twMerge(
+                CompareAccountsPlatformLabelClass({
+                    background: platformType,
+                } as unknown as TCompareAccountsPlatformLabelClassProps)
+            )}
```

Comment:
I think `twJoin` would work here.  An why this `as unknown as TCompareAccountsPlatformLabelClassProps`